webapp/src/languages/flags_validation_tests.rs

```diff
@@ -0,0 +1,25 @@
+use super::static_info::cpp;
+use super::flags_validator::FlagsValidator;
+use super::compiler_flag::CompilerFlag as cf;
+
+#[test]
+fn valid_cpp_flags() {
+    let cpp_info = cpp::construct();
+
+    assert!(FlagsValidator::new().validate(
+        "-std=default", 
+        &cpp_info))
+}
+
+#[test]
+fn prohibited_cpp_flags() {
+    let cpp_info = cpp::construct();
```

Comment:
I see that in both `valid_cpp_flags` and `prohibited_cpp_flags` the cpp_info is constructed the same way and user has no impact on what flags will be used, is this function destined for tests only?

Reply:
cpp::construct() returns information about c++ language. This information is constant, but I'm returning it dynamically because I had problems making it actually ```static```, since it doesn't let to use constructors, and this means no advanced structures for flags storage.
In these tests I wrote flags that were supposed to be tested inside strings in validator calls, ```cpp_info``` is the source that validator uses for its work.